### ISSUE TITLE
error by adding

### ISSUE THREAD

**Issue by Strike2001** [slasktrat/com.tuya#1]
homey v 2.0.0-rc.9 
see picture 
![GitHawk Upload by Strike2001](https://i.imgur.com/vs6Ndbv.jpg)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by slasktrat** [comment]
Hi. Try doing a pull now. But the app also need a tuya account and feeding the key+secret to Homey, have you created an env.json file? Or you could send me you email and I will add you as alpha tester.

**Comment by Strike2001** [comment]
no i did not add this file ( today some smartplugs of the Smart Life series ( Tuya , Woox , and some more companys) and just tried to add it to homey after adding it to the tuya cloud . 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Strike2001** [comment]
my email : m.abels85@googlemail.com 
delete this post  after read 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by slasktrat** [comment]
You are added to alpha channel and will find the app here (need to be logged in) https://apps.athom.com/app/com.tuya.

Please note that there is a bug in the underlying library that requires the app to be restarted between every new device you try to add. I haven't had the time to work around this yet. Also graphics and such is not finished, the app is in early stage.

**Comment by Strike2001** [comment]
so i installed a clean version from the alpha but where must i now put the device id or secret id that the app need to access the  api cloud . 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by slasktrat** [comment]
They are bundled with the app, so you don't need to think about them :)

**Comment by Strike2001** [comment]
every search after new device runs in time out after 30.... 
after the installation of the app from appstore the doesn't ask for any link to the tuya cloud. 
so the app can t find something. a preferences page must be added where it is possible to save this. 
later i test it from github again with the env.json in the main directory created. and install it again maybe this connects to the cloud and showing my sockets. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by slasktrat** [comment]
Are you putting the sockets in pair mode? (Should blink fast while searching for them). If you have already paired them with app store app you need to reset the sockets

**Comment by Strike2001** [comment]
yes , on the searching part the socket react with a on/off click , then the search running to timeout and by press the next button no device are added ,

so the app found the device that shows the reaction of the socket . 

Von meinem iPhone gesendet

**Comment by kshvjsjsb** [comment]
Hi, fantastic job! I have some tuya devices as well. Could I get access as an alpha tester as well?
Happy to contribute my Feedback.

**Comment by Wateboy** [comment]
I also want to be added to alpha to test the smartplugs.

**Comment by slasktrat** [comment]
Need your emails :)

**Comment by Wateboy** [comment]
> Need your emails :)

henke.carlsson@telia.com

**Comment by Strike2001** [comment]
@ version 0.0.4 alpha , added wifi id / password , my wifi socket from Woox found and added to homey successful . great work 👍 , i have a mosiquitto device from tuya too , it can be turn on / off only is it possible too add it too in the future ?

**Comment by slasktrat** [comment]
Glad to hear :) Could you post a link to the socket you use and I will add it to the list of supported devices.

Regarding the mosquitto device, does it show up in the list if you search for it as a socket? (I'm not filtering on any id yet, so all found tuya devices should be listed)

@Wateboy: you are now added to the alpha-tester list :)

**Comment by Strike2001** [comment]
here is the link to the cooperation compamy Woox Home 
the plug R4026 
-- http://www.wooxhome.com/r4026/

the mosquito killer is from the coop. company oittm
-- https://www.oittm.com/Oittm-Electronic-Mosquito-Killer

and another branding names i know : 
Smart Life (own coop app),
io.e (own coop app),
BriHome (own coop app),
luminea (www.pearl.com works under io.e), 
hama (oca),
Kafo (oca),
Jinvoo (oca),
xenon smarthome (oca),
BrightFun (oca),
Lombex (oca),
WStein (oca),
GWHome (oca),
UCSmarthome (oca),
Luminious (oca),
UGSmart (oca),
DingOu Elec (oca),
Meross (oca),
eHomeLife (oca),
AHome (oca),
eFamilyCloud (oca),
Sharper Image (oca),
Delexi ,
and i think that are not all 🤣

**Comment by Wateboy** [comment]
Works great with my NEO WiFi Smart Plugg 16A max 3680w.
I got some delay in the update of powerconsumption in the app, is that something you can select? In tuya app the livefeed is really good and accurate.

**Comment by Strike2001** [comment]
after searching my mos... trap device i tryed to add him , i can successfully add him ass socket device but the on/off button does not work

**Comment by senadaruc** [comment]
Hi, I have a Door Sensor Tuya and i need a beta access, can you please add my email: senad.aruc@gmail.com

Thanks,

**Comment by senadaruc** [comment]
Hello, Can you please add a support for this Door/Windows sensor?

https://www.aliexpress.com/item/Home-Door-Window-Detector-WiFi-App-Notification-Alerts-Battery-Operated-Home-Security-Sensor-Work-with-Alexa/32973283622.html

Thanks,

**Comment by Tuhulka** [comment]
Hi. Good afternoon. Add me for testing. tuhulka@gmail.com

**Comment by Joanniej** [comment]
Hi, great add me for testing. Got some devices as well

**Comment by senadaruc** [comment]
Gents, Did you manage to get into beta testing program?

S,

**Comment by Joanniej** [comment]
No, not  yet did you?

**Comment by Tuhulka** [comment]
No.

**Comment by senadaruc** [comment]
NO!

**Comment by Tuhulka** [comment]
https://ru.aliexpress.com/item/Lonsonho-2-1-Smart-Plug-Wi-Fi-16A/32952681576.html?spm=a2g0s.9042311.0.0.1f6f33edju2L9Z

**Comment by runner780** [comment]
Hi, I like alpha/beta access aswell. Can you add me? My email is rspelten@gmail.com thx in advance!

**Comment by slasktrat** [comment]
You are all invited now :)

**Comment by runner780** [comment]
Thank you, any idea if these will work?

https://s.click.aliexpress.com/e/Cy0IbIY

**Comment by B-maxX** [comment]
@slasktrat 
Hi,
can you add me aswell, please?

belinda@jach-reinke.de

**Comment by mjivani** [comment]
Hi, I'd like to be on the alpha test as well - mohit.jivani@gmail.com

Have a few thermostats that work on the Tuya API

**Comment by Joanniej** [comment]
Hai,

I tried to open the link but  it says it is not there https://apps.athom.com/app/com.tuya.
I am logged in with Athom.

**Comment by JohPePe** [comment]
@slasktrat
Hi,
can you add me aswell, please?
Johan.persson@jpdom.nu
